### PR TITLE
feat(frontend): BtcConvertProgress component

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

This PR contains a basic version of the BtcConvertProgress component. There is already a new, much fancier designs in Figma for this modal, therefore I kept it simple in the first iteration.

Screenshot 2024-11-19 at 12 24 50

### CODE REVIEW

File: src/frontend/src/btc/components/convert/BtcConvertProgress.svelte [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+	import type { ProgressStep } from '@dfinity/gix-components';
+	import InProgressWizard from '$lib/components/ui/InProgressWizard.svelte';
+	import { ProgressStepsConvert } from '$lib/enums/progress-steps';
+	import { i18n } from '$lib/stores/i18n.store';
+
+	export let convertProgressStep: string = ProgressStepsConvert.INITIALIZATION;
+
+	let steps: [ProgressStep, ...ProgressStep[]];
```
Comment (peterpeterparker) [inline src/frontend/src/btc/components/convert/BtcConvertProgress.svelte:9]: @AntonioVentilii somewhere there should be a type for this now right?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

Just tagged Antonio because there is a shorthand type floating around somewhere which later on can replace the type.

File: src/frontend/src/btc/components/convert/BtcConvertProgress.svelte (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/btc/components/convert/BtcConvertProgress.svelte:9]: Just searched through the codebase and I found this - https://github.com/dfinity/oisy-wallet/blob/main/src/frontend/src/lib/types/transaction.ts#L44, which is not really general. So, maybe it's still only planned to create some kind of `NonEmptyArray<T>`?

Comment (peterpeterparker) [reply src/frontend/src/btc/components/convert/BtcConvertProgress.svelte:9]: Sure. Already approved hours ago ;)

**Pull request merged**